requirements.txt

```diff
@@ -4,4 +4,5 @@ flake8-quotes==3.3.1
 flake8-variables-names==0.0.5
 pep8-naming==0.13.2
 django-debug-toolbar==3.2.4
-djangorestframework==3.13.1
\ No newline at end of file
+djangorestframework==3.13.1
+pillow
```

Comment:
you should write version, for example
pillow==10.3.0

Reply:
I deleted the files from the repository because they were uploaded by mistake and do not affect the passing of the tests.   I thought that they are not needed here.